src/server/middleware/access-tokens/TokenServer.js

```diff
@@ -0,0 +1,104 @@
+/*eslint-env node*/
+
+/**
+ * @author lattmann / https://github.com/lattmann
```

Comment:
change the names please even if it is a complete copy

Reply:
darn. It was automatically code folded and I missed it! Thanks

Reply:
Good catch. Code folding burned me here :)